packages/components/tests/dummy/app/components/dummy-footer.hbs

```diff
@@ -1,45 +1,42 @@
 <footer class="dummy-footer">
 
-  <div class="dummy-row dummy-row--first">
-    <div class="dummy-content-width-wrapper">
-      <p class="dummy-title-small">Our Accessibility Statement</p>
-      <p class="dummy-paragraph">
-        Each of our components intends to conform to
+  <div class="dummy-footer__accessibility">
+    <p class="dummy-title-small">Our Accessibility Statement</p>
+    <p class="dummy-paragraph">
+      Each of our components intends to conform to
+      <a
+        href="https://www.w3.org/WAI/standards-guidelines/wcag/"
+        class="dummy-link"
+        target="_blank"
+        rel="noopener noreferrer"
+      >WCAG 2.1 AA standards</a>
+      (at a minimum). If you find an issue with any component, you can let us know in any of the following ways:
+    </p>
+    <ol class="dummy-list">
+      <li>
+        Post a message in the
         <a
-          href="https://www.w3.org/WAI/standards-guidelines/wcag/"
+          href="https://app.slack.com/client/T024UT03C/C7KTUHNUS/thread/CPB8GS9QT-1638823666.137000"
           class="dummy-link"
           target="_blank"
           rel="noopener noreferrer"
-        >WCAG 2.1 AA standards</a>
-        (at a minimum). If you find an issue with any component, you can let us know in any of the following ways:
-      </p>
-      <ol class="dummy-list">
-        <li>
-          Post a message in the
-          <a
-            href="https://app.slack.com/client/T024UT03C/C7KTUHNUS/thread/CPB8GS9QT-1638823666.137000"
-            class="dummy-link"
-            target="_blank"
-            rel="noopener noreferrer"
-          >#team-design-systems</a>
-          channel in our internal Slack
-        </li>
-        <li>
-          Email
-          <a href="mailto:design-systems@hashicorp.com">
-            design-systems@hashicorp.com
-          </a>
-        </li>
-        <li>
-          File an issue on this
-          <a href="https://github.com/hashicorp/design-system" target="_blank" rel="noopener noreferrer">GitHub
-            repository</a>
-        </li>
-      </ol>
-    </div>
+        >#team-design-systems</a>
+        channel in our internal Slack
+      </li>
+      <li>
+        Email
+        <a href="mailto:design-systems@hashicorp.com">
+          design-systems@hashicorp.com
+        </a>
+      </li>
+      <li>
+        File an issue on this
+        <a href="https://github.com/hashicorp/design-system" target="_blank" rel="noopener noreferrer">GitHub repository</a>
+      </li>
+    </ol>
   </div>
-  <div class="dummy-row dummy-row--last">
-    <p class="dummy-paragraph dummy-content-width-wrapper">
+  <div class="dummy-footer__github">
```

Comment:
[suggestion] everything else is contained or centered but this one isn't; would be nice for it to match.

Reply:
Yes, I like it! Pushed the suggested change.